src/Hpack/Syntax/DependencyVersion.hs

```diff
@@ -5,10 +5,13 @@ module Hpack.Syntax.DependencyVersion (
 , GitRef
 , GitUrl
 , DependencyVersion(..)
-, SourceDependency(..)
 , dependencyVersionFromCabal
+, SourceDependency(..)
+, sourceDependencyFromObject
 
 , scientificToVersion
+, scientificToDependencyVersion
+, parseVersionRange
```

Comment:
Might think about re-organising `H.S.Dependency` and `H.S.DependencyVersion` now; there's not a whole lot of logic about what's going in which module left.